charter.md

```diff
@@ -0,0 +1,16 @@
+## Membership Committee Charter
+**Approved:** April 2017  
+
+### Purpose
+
+The Nazareth Membership Committee is responsible for creating and maintaining structures that manage the membership of the Nazareth campus.
+
+### Remit
+
+- Create and maintain membership register
+- Create and maintain membership application form
+- Regularly notify course participants of membership eligibility, and prepare applications for voting at business meeting
+- Regularly review the [Complaints Policy](https://github.com/foundersandcoders/legal/blob/cc4f6eb37ba05cd422ab49d175140147a306ab7d/complaints.md), modifying it to suit the needs of the Nazareth campus if necessary
```

Comment:
any reason why this link references a commit instead of the master branch? 

maybe it should be:

https://github.com/foundersandcoders/legal/blob/master/complaints.md